src/views/SingleEventView.vue

```diff
@@ -168,8 +168,8 @@ export default {
     AnnouncementModal,
   },
   props: {
-    eventId: { // id is a number, but props are always passed as strings
-      type: String,
+    eventId: {
+      type: Number,
       required: true,
```

Comment:
I added a method to the router to turn this prop into a Number so we don't have to mess with parsing later on